annotation/app/tasks/services.py

```diff
@@ -651,3 +652,29 @@ def save_agreement_metrics(
     ]
     db.bulk_save_objects(metrics)
     db.commit()
+
+
+def get_task_index_in_tasks_list_by_task_id(
+    task_id: int, tasks_list: List[ManualAnnotationTaskSchema]
+) -> int:
+    list_of_task_ids = [task.id for task in tasks_list]
+    return list_of_task_ids.index(task_id)
+
+
+def get_previous_task(
+    tasks_list: List[ManualAnnotationTaskSchema], current_index: int
+) -> Optional[ManualAnnotationTaskSchema]:
+    previous_task_index = current_index - 1
+    if previous_task_index < 0:
```

Comment:
Let's simplify: 
If we order by id, so previous task is:
`select .. where id < current_id order by id desc limit 1`
and next is
`select ... where id > current_id order by id asc limit 1`

Reply:
Rewritten in a suggested way